build/vagrant/infra/userdata.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -eo
+
+# update machine
+sudo apt-get update
+
+# install needed tools
+sudo apt-get install awscli vagrant python3-pip -y
+sudo apt-get install virtualbox virtualbox-guest-additions-iso -y
+pip3 install awscli --upgrade --user
+
+#install packer
+sudo apt-get install wget -y
+wget "https://releases.hashicorp.com/packer/${PACKER_VERSION}/packer_${PACKER_VERSION}_linux_amd64.zip"
+unzip "packer_${PACKER_VERSION}_linux_amd64.zip"
+sudo mv packer /usr/local/bin/
+packer -v
+
+# download spel
+cd /tmp
+git clone https://github.com/plus3it/spel.git
```

Comment:
Can we get the commit being built from the codebuild job, and check it out here?

Reply:
Running this in codebuild right now to ensure that change worked